tools/realsense-viewer/realsense-viewer.cpp

```diff
@@ -336,7 +336,7 @@ int main(int argc, const char** argv) try
         ImGui::PushStyleVar(ImGuiStyleVar_WindowPadding, ImVec2(5, 5));
         ImGui::SetNextWindowPos({ 0, viewer_model.panel_y });
 
-        std::string add_source_button_text = to_string() << " " << textual_icons::plus_circle << "  Add Source\t\t\t\t\t\t\t\t\t\t\t";
+        std::string add_source_button_text = to_string() << " " << textual_icons::plus_circle << "  Add Source (" << (device_names.size() - device_models->size()) << " available)\t\t\t\t\t\t\t\t\t\t\t";
```

Comment:
That is a very nice addition for user experience.
To make an impact I'll look if and how it is possible to convert it into a badge with a notification to make it less verbose and avoid `Add Source (0 available)` message
![image](https://user-images.githubusercontent.com/17897861/72972957-02a64180-3dd5-11ea-9e22-a9549a1bc7a4.png)

Reply:
I suggest merging this now, and then changing the text to an icon in a second PR.   I, just this minute, had another person claim that we don't support multiple T265's when we do, but they couldn't tell because of the reasons outlined above.

Reply:
Hi @radfordi - Thanks a lot for your PR.
It needs to be rebased to our latest development branch - easiest way is to make a new branch starting from our development branch and change the line again there.
Can you do that please?